src/custom/utils/analytics/listEvents.ts

```diff
@@ -0,0 +1,36 @@
+import { Category, _reportEvent } from './index'
+
+type UpdateListAction = 'Popup' | 'List Select'
```

Comment:
This is perfect, altough nitpicking `Popup` is not an action, its a location from which you take the action.
`UpdateListLocation`?